modules/kubernetes/aks-core/namespace.tf

```diff
@@ -99,7 +99,7 @@ resource "kubernetes_network_policy" "tenant" {
       }
       to {
         ip_block {
-          cidr = "169.254.25.10/32"
```

Comment:
Strange that this have worked. Since it does my guess is that it's okay to delete. I will verify this.

Reply:
I think we can leave this for now and we can verify that it's okay to remove this block all together.